web_src/js/features/citation.js

```diff
@@ -40,28 +40,35 @@ export async function initCitationFileCopyContent() {
     $citationCopyApa.toggleClass('primary', !isBibtex);
   };
 
-  try {
-    await initInputCitationValue($citationCopyApa, $citationCopyBibtex);
-  } catch (e) {
-    console.error(`initCitationFileCopyContent error: ${e}`, e);
-    return;
-  }
-  updateUi();
+  $('#cite-repo-button').on('click', async (e) => {
+    const dropdownBtn = e.target.closest('.ui.dropdown.button');
+    dropdownBtn.classList.add('is-loading');
```

Comment:
Sidenote: for the worry about "some refactoring may break this logic", I have 2 ideas for it:

1. Just leave it as-is. Because we have error-catch mechanism, so if it breaks, there should be immediate error reports on the UI (`dropdownBtn.xxx` will fail if it is null)
2. Add a helper function like `closestVisibleElem()`, maybe it could be useful for some more cases.

Reply:
`.closest` is straightforward, I think such a helper would only very rarely be useful.